### PR TITLE
Added tests for course.page.choice

### PR DESCRIPTION
**Pull request by dzhuang**

### CODE REVIEW

File: course/page/choice.py [hunk 1]
```diff
@@ -387,7 +404,7 @@ def normalized_bytes_answer(self, page_context, page_data, answer_data):
         permutation = page_data["permutation"]
 
         if answer_data is None:
-            unpermuted_choice = None
+            return None
```
Comment (dzhuang) [inline course/page/choice.py]: I modified here so that page submitted with no answer won't generate a json file.

File: course/page/choice.py [hunk 2]
```diff
@@ -669,7 +695,7 @@ def normalized_bytes_answer(self, page_context, page_data, answer_data):
         permutation = page_data["permutation"]
 
         if answer_data is None:
-            unpermuted_choices = None
+            return None
```
Comment (dzhuang) [inline course/page/choice.py]: Same as above.

**Comment by dzhuang** [comment]
Done.

**Comment by dzhuang** [comment]
@inducer I think this is ready.

**Comment by inducer** [comment]
LGTM, thx!

**Pull request merged**